### PR TITLE
Harvester for CogPrints and CaltechAuthors

### PR DESCRIPTION
**Pull request by GrantRVD**
Adds OAIHarvester for the [Cognitive Sciences ePrint Archive](http://cogprints.org/cgi/oai2?verb=ListRecords&metadataPrefix=oai_dc)

### CODE REVIEW

File: scrapi/harvesters/cogprints.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+from __future__ import unicode_literals
+
+from scrapi.base import OAIHarvester
+
+
+class CogPrintsHarvester(OAIHarvester):
+    short_name = 'cogprints'
+    long_name = 'Cognitive Sciences ePrint Archive'
+    url = 'http://www.http://cogprints.org/'
```
Comment (fabianvf) [inline scrapi/harvesters/cogprints.py]: It looks like you may have an extra http in here

Comment (fabianvf) [inline scrapi/harvesters/cogprints.py]: Other than that, looks great! Thanks!

**Comment by GrantRVD** [comment]
Thanks @fabianvf. Sorry about the typo. It's been fixed. Let's see what we get now.

**Comment by erinspace** [comment]
Thanks so much for this PR! Would you mind adding a favicon for cogprints into the https://github.com/fabianvf/scrapi/tree/develop/img/favicons folder? I think we forgot that step in the instructions! Will do that asap.

The favicon should be named 'cogprints_favicon.ico'

**Comment by GrantRVD** [comment]
Sure thing, @erinspace. I've added the favicon, and I've repeated the process for the caltech repository. Meant to submit that as a separate pull request, but the commits were integrated with this one, so I'll wait.

**Comment by erinspace** [comment]
Awesome, thank you @GrantRVD ! @chrisseto @fabianvf take a look!

File: scrapi/harvesters/caltech.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+from __future__ import unicode_literals
+
+from scrapi.base import OAIHarvester
+
+
+class CaltechHarvester(OAIHarvester):
+    short_name = 'caltech'
+    long_name = 'CaltechAUTHORS'
+    url = 'http://authors.library.caltech.edu//'
```
Comment (chrisseto) [inline scrapi/harvesters/caltech.py]: Should this url only have a single slash at the end?

**Comment by fabianvf** [comment]
Looks great. Once that extra ```/``` is taken care of, I think this is ready for a merge. Thanks for hacking with us! 
Post-merge you'll be able to view the harvested documents at osf.io/share

**Comment by GrantRVD** [comment]
Sorry for the delays! Thanks @chrisseto for pointing out the extra slash. Will take care of it now.

**Pull request merged**